R/get_pro.R

```diff
@@ -5,77 +5,96 @@
 #' #'
```

Comment:
Would be great with a description of the `status` parameter options that can be provided, so users know what the options are.